sql/core/src/test/scala/org/apache/spark/sql/execution/benchmark/DataSourceReadBenchmark.scala

```diff
@@ -147,6 +159,16 @@ object DataSourceReadBenchmark extends SqlBasedBenchmark {
           }
         }
 
+        sqlBenchmark.addCase("SQL Parquet Vectorized (Delta Binary)") { _ =>
```

Comment:
Can we:
- move this up to be adjacent to the Parquet cases
- change the name to "SQL Parquet Vectorized Format v2" or something? 

You also need to rebase and handle the newly added `BooleanType` here.

Reply:
I guess I didn't understand the implication when you suggested I need to rebase!
It looks like the Boolean type breaks vectorized reading. The boolean column gets encoded as RLE but the vectorized RLE reader does not support Boolean.
```
[info] Caused by: java.lang.UnsupportedOperationException: only readInts is valid.
[info] 	at org.apache.spark.sql.execution.datasources.parquet.VectorizedRleValuesReader.readBooleans(VectorizedRleValuesReader.java:368)
```
I guess I could fix this. Would you recommend a different PR, or should I make the change in this one?

Reply:
Right, Boolean is encoded as RLE in Parquet format V2, so we'd need to handle it too. I think we should raise a separate PR for this.

For now, can you remove the changes in `DataSourceReadBenchmark` and split it out into a separate PR later once we have the boolean type supported?